backport/backport.ts

```diff
@@ -180,17 +180,21 @@ const backportOnce = async ({
 	}
 }
 
-const getFailedBackportCommentBody = ({
+export const getFailedBackportCommentBody = ({
 	base,
 	commitToBackport,
 	errorMessage,
 	head,
+	title,
 }: {
 	base: string
 	commitToBackport: string
 	errorMessage: string
 	head: string
+	title: string
 }) => {
+	const backportMilestone = base.startsWith('v') ? base.substring(1) : base
+	const escapedTitle = title.replaceAll('"', '\\"')
```

Comment:
We need to add the branch name to the title as well `[vX.Y.Z]`

Reply:
Doesn't happen for failing PRs I think :/

Reply:
Hm... the title is templated in https://github.com/grafana/grafana-github-actions/pull/150/files#diff-4ba6a76ffc49abf7b860d7951507573e42bc5a66d7fe0480297b287ad8c2fdc8R342 and the call to `getFailedBackportCommentBody` happens after that so I think this should work 🙂

Reply:
Tested on a private repo with settings similar to what we have in grafana/grafana. I've added the generated output to the description, so it seems to work :)